electron-app/src/services/uiconfig.ts

```diff
@@ -37,6 +42,15 @@ export default class UiConfig {
         return configData;
       }
 
+      //* Check for mac if it's on old config location
+      if (getPlatform() === 'mac' && checkPathExists(CONFIG_FILE_PATH_LEGACY)) {
+        const existingConfigData = this.getExistingConfig(
```

Comment:
Can we delete the old config after copying the component and also can we check whether we have copied the old config or not at first place

Reply:
I prefer to leave the old config on it's current location. I want to avoid auto deleting files which might confuse some users. For the second point, it will be already handled by this condition. The first condition checks if the config is existing on the directory, if not, it will copy the old config to the new directory. On succeeding runs, since it's using the new directory config, it won't copy the old one anymore.